.github/workflows/registry.yml

```diff
@@ -87,23 +91,30 @@ jobs:
       - name: Tag helm chart image
         run:  |
           version=${{ steps.get_version.outputs.VERSION }}
-          sed -i "s/latest/$version/g" charts/vela-core/values.yaml
+          sed -i "s/latest/$version/g" $HELM_CHART/values.yaml
           number=${version#"v"}
-          sed -i "s/0.1.0/$number/g" charts/vela-core/Chart.yaml
+          sed -i "s/0.1.0/$number/g" $HELM_CHART/Chart.yaml
+          sed -i "s/0.1.0/$number/g" $LEGACY_HELM_CHART/Chart.yaml
+      - name: Prepare legacy chart
+        run: |
+          rsync -r $LEGACY_HELM_CHART $HELM_CHARTS_DIR
+          rsync -r $HELM_CHART/* $LEGACY_HELM_CHART --exclude=Chart.yaml --exclude=crds
```

Comment:
move it before `tag helm chart image`, that can be easily maintained

Reply:
I moved it to the end of `tag helm chart image`. But your suggestion is also right.